ts/features/bonus/cgn/utils/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const EYCA_DISCOUNTS_URL = "https://eyca.org/discounts/it";
```

Comment:
This can be composed from the EYCA_WEBSITE_URL.

Reply:
That's absolutely better, thank you. 🙏

Done as you suggested with this commit: [122d472](https://github.com/pagopa/io-app/pull/4689/commits/122d472cd96d6010c134351178fba3259c60a7d2)